src/commands/test/run/uitest.ts

```diff
@@ -119,4 +125,47 @@ export default class RunUITestsCommand extends RunTestsCommand {
   protected getSourceRootDir() {
     return this.buildDir;
   }
+
+  protected async mergeTestArtifacts(): Promise<void> {
+    if (!this.mergeNUnitXml) {
+      return;
+    }
+
+    let tempPath: string = await pfs.mkTempDir("appcenter-uitestreports")
+    let reportPath: string = path.join(this.generateReportPath(), "nunit_xml_zip.zip");
```

Comment:
What happens here if the `generateReportPath()` returns `null` or a non existent path?

Reply:
Also - can we check this on Windows to make sure it behaves the same?